tests/framework/clients/k3d/k3d.go

```diff
@@ -89,8 +90,12 @@ func CreateAndImportK3DCluster(client *rancher.Client, name string) (*apisV1.Clu
 		return nil, errors.Wrap(err, "CreateAndImportK3DCluster: failed to create k3d cluster")
 	}
 
+	kubeProvisioningClient, err := client.GetKubeAPIProvisioningClient()
+	if err != nil {
+		return nil, errors.Wrap(err, "CreateAndImportK3DCluster: failed to instaniated kube api provisioning client")
```

Comment:
typo on instantiate?

Reply:
Fixed, and I sent you the freeform job as well Caleb